rtl/vendor/pulp_platform_fpnew.lock.hjson

```diff
@@ -8,7 +8,7 @@
 {
   upstream:
   {
-    url: https://github.com/pulp-platform/fpnew.git
-    rev: 11659d7ff3580ac3226c6d56a90ef717cdc530e3
+    url: https://github.com/davideschiavone/fpnew.git
```

Comment:
to be changed once https://github.com/openhwgroup/cvfpu/pull/93/files has been merged